src/graphql/status.rs

```diff
@@ -180,8 +180,8 @@ impl ConfigMutation {
         let is_local = ctx.data::<bool>()?;
 
         if *is_local {
-            warn!("Config is local");
-            return Ok(false);
+            warn!("Configuration set GraphQL API request disregarded: Giganto is running with a local configuration, so the request to set the configuration is disregarded.");
```

Comment:
I suggest the following to clarify the message:

The request to the GraphQL API to change the configuration was ignored because this is running with a local configuration.

Reply:
Done